cmd/juju/commands/bootstrap_clouds.go

```diff
@@ -45,7 +45,9 @@ func formatCloudDetailsTabular(clouds cloudList, credStore jujuclient.Credential
 		for _, name := range cloudNames {
 			cred, err := credStore.CredentialForCloud(name)
 			if err != nil && !errors.IsNotFound(err) {
-				return errors.Annotatef(err, "error loading credential for cloud %q", name)
+				ctx.Warningf("error loading credential for cloud %v: %v", name, err)
+				p(name, "ERROR", "")
```

Comment:
I think INVALID might be better
I'm not sure how logging message will interact with printing the tabular output. You normally see log messages when --show-log is used.
We need new unit tests for tabular (and yaml/json) output of list-credentials or bootstrap --cloud etc etc when there's a bad credential.